Makefile

```diff
@@ -36,8 +36,8 @@ static-code-analysis: vendor ## Runs a static code analysis with phpstan/phpstan
 .PHONY: static-code-analysis-baseline
 static-code-analysis-baseline: vendor ## Generates a baseline for static code analysis with phpstan/phpstan and vimeo/psalm
 	mkdir -p .build/phpstan
-	echo '' > phpstan-baseline.neon
-	vendor/bin/phpstan --configuration=phpstan.neon --error-format=baselineNeon --memory-limit=-1 > phpstan-baseline.neon || true
+	echo "parameters:\n\tignoreErrors: []\n" > phpstan-baseline.neon
+	vendor/bin/phpstan --configuration=phpstan.neon --generate-baseline=phpstan-baseline.neon --memory-limit=-1 || true
```

Comment:
FYI, next phpstan version will come with a `--allow-empty-baseline` option (see https://github.com/phpstan/phpstan-src/pull/776).

Reply:
Thank you, @Lctrs, that is great to hear!